libs/CMakeLists.txt

```diff
@@ -90,7 +91,7 @@ if(HPX_WITH_TESTS)
   endif()
 
   if (HPX_WITH_TESTS_HEADERS)
-    add_hpx_pseudo_target(tests.headers.modules)
+    add_custom_target(tests.headers.modules)
```

Comment:
This will break things on Windows. Please stick to using `add_hpx_pseudo_target`.

Reply:
I thought I had only moved code around. If I introduced something new and bad in the cmake, I'd better fix it ...

Reply:
this appear to have been introduced by a change in the library generator code in `components/create_component_skeleton.py` - I have removed it from CMakeLists in this PR and will push shortly.

Reply:
Could you please fix the generator script in this case as well?